bigtable/google/cloud/bigtable/client.py

```diff
@@ -76,8 +78,14 @@ def _make_data_stub(client):
     :returns: A gRPC stub object.
     """
     if client.emulator_host is None:
+        # NOTE: 'grpc.max_message_length' will no longer be recognized in
+        # grpcio 1.1 and later.
+        max_msg_length = (('grpc.max_message_length', _MAX_MSG_LENGTH_100MB),
+                          ('grpc.max_receive_message_length',
+                           _MAX_MSG_LENGTH_100MB))
```

Comment:
@daspecster You missed @nathanielmanistaatgoogle feedback that `max_msg_length` should **also** be a non-public module-level constant.